### DISCUSSION TITLE
envbox hangs on process: sysbox-fs

### DISCUSSION THREAD

**Discussion by bcpereira12** [coder/coder#8951] [General]
I'm trying to configure envbox in our kubernetes cluster to test the docker-in-docker functionality. To create workspaces in our Coder OSS, an internal CA is required. To accomplish this, I have created a docker image that uses the base image for envbox and I add our internal CA. I am using the [envbox starter template](https://github.com/coder/coder/tree/main/examples/templates/envbox) and have set the container image to my custom docker image. 

When creating the workspace, it hangs on "Envbox startup complete!", but the kubernetes logs shows 

`{"ts":"2023-08-07T22:45:15.52184497Z","level":"ERROR","msg":"child log","caller":"/home/runner/work/envbox/envbox/background/process.go:248","func":"github.com/coder/envbox/background.scanIntoLog","logger_names":["sysbox-fs"],"fields":{"process":"sysbox-fs","content":"time=\"2023-08-07 22:45:15\" level=error msg=\"Failed to write to /proc/sys/net/core/default_qdisc: open /proc/sys/net/core/default_qdisc: no such file or directory\""}}
`

Thank you!

**Comment by jamestoyer** [comment]
@bcpereira12 did you ever work out why this was happening?
